### PR TITLE
Add to docs docs configuring file format options

### PR DESCRIPTION
**Pull request by VioletM**

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<pre>
+[normalize]{'\n'}
+loader_file_format="{props.file_type}"
+</pre>
+
+3. You can set the `loader_file_format` via ENV variable:
+
+```
+NORMALIZE__LOADER_FILE_FORMAT
+```
```
Comment (rudolfix) [inline docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx:23]: now we have 4th method which is also cool: you can set it on a resource
https://dlthub.com/devel/general-usage/resource#pick-loader-file-format-for-a-particular-resource

File: docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import CodeBlock from '@theme/CodeBlock';
+
+<span>There are several ways of configuring dlt to use <code>{props.file_type}</code> file format for normalization step and to store your data at the destination:</span>
+
+1. <span>You can set the <code>loader_file_format</code> argument to <code>{props.file_type}</code> in the run command:</span>
+
+<pre>
```
Comment (rudolfix) [inline docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx:7]: no way to use language tags here?
```py
```
not rendered correctly? the problem is that this won't be linted by our tooling

File: docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx (hunk 1 above)
Comment (VioletM) [reply docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx]: Oh, thanks for pointing this out, added!

File: docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx (hunk 2 above)
Comment (VioletM) [reply docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx]: Added language tags. 

Sorry, I had to play around with tags to be able to use props, so I just forgot about this.

File: docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+loader_file_format="{props.file_type}"
+</pre>
+
+3. You can set the `loader_file_format` via ENV variable:
+
+<pre language="sh">
+export NORMALIZE__LOADER_FILE_FORMAT="{props.file_type}"
+</pre>
+
+4. You can set the file type directly in [the resource definition](../../general-usage/resource#pick-loader-file-format-for-a-particular-resource).
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx:25]: is it possible to add the code-snippet from the resource page? like:
```
@dlt.resource(file_format="{props.file_type}")
def generate_rows(nr):
    for i in range(nr):
        yield {'id':i, 'example_string':'abc'}
```
so readers don't have to go to another page

Comment (VioletM) [reply docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx:25]: I'm not sure if our snippet checker would be happy with it

```
<pre language="py">
@dlt.resource(file_format="{props.file_type}"){'\n'}
def generate_rows(nr):{'\n'}
&nbsp;&nbsp;&nbsp;&nbsp;for i in range(nr):{'\n'}
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;yield &#123;'id':i, 'example_string':'abc'&#125;
</pre>
```

**Review by AstrakhantsevaAA — APPROVED** [review]
Perfecto!

**Pull request merged**